src/test/java/org/apache/commons/lang3/reflect/TypeUtilsTest.java

```diff
@@ -543,221 +539,108 @@ public void testIsArrayTypeClasses() {
     @Test
     public void testIsAssignable() throws SecurityException, NoSuchMethodException,
             NoSuchFieldException {
-        List list0 = null;
```

Comment:
I don't understand at all what rhe original author was doing here. This is weird.